src/lib/EasyEdit.jsx

```diff
@@ -32,7 +32,7 @@ export default class EasyEdit extends React.Component {
   }
 
   isNullish(value) {
```

Comment:
A good name for a function that checks for null, undefined, or an empty string could be something like isNullOrUndefinedOrEmpty to make its purpose explicit. 

If you prefer an even shorter method name, you can use something like isNUE as an abbreviation for "isNull, Undefined, or Empty." However, remember that shorter names may sacrifice some readability and clarity.

Reply:
done